src/Sarif.UnitTests/Processors/GenericTests/GenericProcessorTests.cs

```diff
@@ -14,7 +14,7 @@ public class GenericProcessorTests
         private List<int> GenerateRandomIntList()
         {
             Random r = new Random();
-            int size = r.Next(100);
+            int size = r.Next(1, 100);
```

Comment:
Wait a minute, we have a unit test that depends on throwing a random number? That's not kosher: unit tests should be absolutely repeatable. After you merge this, would you please take just a couple of minutes, look at this test, and see if it can be written in a deterministic way? (Just a couple of minutes!) @michaelcfanning FYI,

Reply:
Sure thing. And I know this isn't the only occurrence, I fixed the same problem in another test a couple of weeks ago.

Reply:
In that case, please just file a bug on the SDK: "Unit tests should not throw random numbers", and we can address it comprehensively later. I'm sure @michaelcfanning wouldn't want us spending any significant amount of time on this now.

Reply:
OK.